package.json

```diff
@@ -26,5 +26,9 @@
     "webpack-cli": "^4.5.0",
     "webpack-dev-server": "^3.11.2",
     "webpack-merge": "^5.7.3"
+  },
+  "dependencies": {
+    "http-server": "^0.12.3",
+    "style-loader": "^2.0.0"
```

Comment:
do we need these?

Reply:
we need style-loader, i removed http-server tho